src/modules/recipes/models.ts

```diff
@@ -0,0 +1,82 @@
+// Recipe için TypeScript türü
+export interface Recipe {
+  idMeal: string;
+  strMeal: string;
+  strMealThumb: string;
+  strCategory?: string;
+  strArea?: string;
+  strInstructions?: string;
+  strIngredient1?: string;
+  strIngredient2?: string;
+  strIngredient3?: string;
+  [key: string]: any; // Ek alanlar için
+}
+
+export type FormData = {
```

Comment:
This model should be local to the NewRecipeForm component, it should not be needed in multiple places in the application.